### PR TITLE
fix: ANDAUTH-22 removed .idea folder from repository

### PR DESCRIPTION
**Pull request by pa1pal**
As it was IDE settings folder, contains system generated files, need not pushed to git repository. 
[Issue link](http://jira.xwiki.org/browse/ANDAUTH-22)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,6 @@
 /.idea/libraries
 .DS_Store
 /build
+.idea/
+android-authenticator.iml
```
Comment (sdumitriu) [inline .gitignore:8]: You could just say `*.iml`

File: app/app.iml [hunk 2]
```diff
@@ -9,7 +9,6 @@
     <facet type="android" name="Android">
```
Comment (sdumitriu) [inline app/app.iml:9]: This file and the next could be removed as well, right?

**Comment by vmassol** [comment]
Regarding the ignore file, note that we already have several existing ones. For example: https://github.com/xwiki/xwiki-platform/blob/master/.gitignore

File: .gitignore (hunk 1 above)
Comment (pa1pal) [reply .gitignore:8]: Yes. this would be much better and include all files. I will include this in my next PR.

File: app/app.iml (hunk 2 above)
Comment (pa1pal) [reply app/app.iml:9]: Yes.

**Pull request merged**